scss/_buttons.scss

```diff
@@ -274,6 +274,7 @@
       &:active {
```

Comment:
While I was testing the PR, I tried to add `.active` on those buttons and their rendering is not correct:
![Screenshot from 2022-04-01 12-17-29](https://user-images.githubusercontent.com/17381666/161244629-4ee48ca6-8718-43e9-b7aa-3e59732d27bb.png)

We could improve the selector by doing:

```scss
&:active,
&.active {
```

Reply:
That was not in the PR ;-) but ok :-)